### ISSUE TITLE
New Release?

### ISSUE THREAD

**Issue by daemonspudguy** [Ordim3n/That-Shortcut-Thingy#5]

**Comment by MiksFish** [comment]
Yeah, Godmode9 has a new release (6th anniversary 2.1.1) and a new prerelease (9th anniversary 2.2.0), so this shortcut release is actually pretty outdated lol
But like really, make a new release, can’t even install cias with this version.

**Comment by daemonspudguy** [comment]
This project is dead. The issue is going to be closed.

**Comment by MiksFish** [comment]
Wait then why isn’t archived? (New one is A9SP)
